packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -319,15 +320,15 @@ const IdvFailed = ({
                     })}
                 >
                     <FormBody className='form-body' scroll_offset={setScrollOffset()}>
-                        <Text size={is_mobile ? 'xs' : 's'} weight='bold' align='center'>
+                        <Text size={isMobile ? 'xs' : 's'} weight='bold' align='center'>
```

Comment:
In this case, the tablet view will have fontsize of desktop screens. Is this expected?

Reply:
Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.